unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/FriendsHUD/Scripts/FriendsHUDComponentView.cs

```diff
@@ -147,31 +162,61 @@ public void Hide()
                 break;
             case FriendshipAction.APPROVED:
                 friendRequestsTab.Remove(userId);
-                friendsTab.Enqueue(userId, friendEntryModel);
+                friendsTab.Enqueue(userId, model);
                 break;
             case FriendshipAction.REJECTED:
                 friendRequestsTab.Remove(userId);
+                friendsTab.Remove(userId);
                 break;
             case FriendshipAction.CANCELLED:
                 friendRequestsTab.Remove(userId);
+                friendsTab.Remove(userId);
                 break;
             case FriendshipAction.REQUESTED_FROM:
-                friendRequestsTab.Set(userId, friendEntryModel, true);
+                friendRequestsTab.Enqueue(userId, new FriendRequestEntryModel(model, true));
+                friendsTab.Remove(userId);
                 break;
             case FriendshipAction.REQUESTED_TO:
-                friendRequestsTab.Set(userId, friendEntryModel, false);
+                friendRequestsTab.Enqueue(userId, new FriendRequestEntryModel(model, false));
+                friendsTab.Remove(userId);
                 break;
             case FriendshipAction.DELETED:
                 friendRequestsTab.Remove(userId);
                 friendsTab.Remove(userId);
                 break;
             default:
-                Debug.LogError($"UpdateFriendshipStatus not supported: {friendshipAction}");
+                Debug.LogError($"FriendshipAction not supported: {friendshipAction}");
                 break;
         }
     }
 
-    public void Search(string userId) => FilterFriends(userId);
+    public void Set(string userId, FriendshipStatus friendshipStatus, FriendEntryModel model)
+    {
+        Debug.Log($"Set {userId}, {friendshipStatus}, {model.userName}");
```

Comment:
We want to leave this debug log here for some reason?

Reply:
no! thank you for checking it, it was a debugging thing